### ISSUE TITLE
mount should not require fstype/src for state=unmounted

### ISSUE THREAD

**Issue by amenonsen** [ansible/ansible-modules-core#1889]
Issue: Bug report
Ansible version: devel
Ansible configuration: NA
Environment: NA
Summary:

The `mount` module requires that `src` and `fstype` always be specified, even when this information is not needed, as is the case with `state=unmounted`.

Steps to reproduce:

```
- mount: name=/some/mount/path state=unmounted
```

Expected results: Works
Actual results: Doesn't

**Comment by angystardust** [comment]
It seems this issue is fixed now in https://github.com/ansible/ansible-modules-core/pull/1357

**Comment by ansibot** [comment]
@grossws, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by bmillemathias** [comment]
I can confirm this is not fixed in 2.1.x

**Comment by ansibot** [comment]
@grossws, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by grossws** [comment]
@bmillemathias, as I said in #1357, it was merged into `devel` but not to 2.1.x releases.

@ansibot, resolved_by_pr #1357
